docs/api-usage/authentication.mdx

```diff
@@ -142,3 +142,268 @@ mutation {
+    }
+  }
+}
+```
+
+### Refreshing tokens
+
+The [`externalRefresh`](api-reference/mutations/external-refresh.mdx) mutation will generate a new access token when given a valid refresh token.
+
+As with the previous calls, you will need to pass `"mirumee.authentication.openidconnect"` as the `pluginId` argument, this time including the `refreshToken`:
+
+```graphql
+mutation {
+  // highlight-start
+  externalRefresh(
+    pluginId: "mirumee.authentication.openidconnect"
+    input: "{\"refreshToken\": \"xJc...Uas\"}"
+  ) {
+  // highlight-end
+    token
+    refreshToken
+    accountErrors {
+      field
+      message
+      code
+    }
+  }
+}
+```
+
+### Verifying tokens
+
+To verify the token, use the following [`externalVerify`](api-reference/mutations/external-verify.mdx) mutation.
+
+Once again, you will need to pass `"mirumee.authentication.openidconnect"` as the `pluginId` argument, this time including the `token`:
+
+```graphql
+mutation {
+  // highlight-start
+  externalVerify(
+    pluginId: "mirumee.authentication.openidconnect"
+    input: "{\"token\": \"eyJ0eXAiOiJK...J1M7tqSzP0\"}"
+  ) {
+  // highlight-end
+    isValid
+    user {
+      userPermissions {
+        code
+        name
+      }
+    }
+  }
+}
+```
+
+The `user` field will contain the user's details if the token is valid.
+
+### Logging out
+
+Logging out requires the user to be redirected to the identity provider's logout page.
+
+You can prepare the logout URL by calling the [`externalLogout`](api-reference/mutations/external-logout.mdx) mutation. Any parameters you pass in the `input` argument will be included in the logout URL.
+
+```graphql
+mutation ($input: JSONString!) {
+  // highlight-start
+  externalLogout(
+    pluginId: "mirumee.authentication.openidconnect"
+    input: "{\"returnTo\": \"https://example.com\"}"
+  ) {
+  // highlight-end
+    logoutData
+    accountErrors {
+      field
+      message
+      code
+    }
+  }
+}
+```
+
+The `logoutData` field will contain the logout URL:
+
+```json
+{
+  "data": {
+    "externalLogout": {
+      // highlight-next-line
+      "logoutData": "{\"logoutUrl\": \"https://saleor-test.eu.auth0.com/v2/logout?returnTo=https%3A%2F%2Fexample.com\"}",
+      "accountErrors": []
+    }
+  }
+}
+```
+
+Next, redirect the user to the identity provider's logout page.
+
+## App authentication
+
+Unlike regular users, Saleor Apps have long-living tokens they can use to communicate with the API. The tokens are assigned at the App installation time and must be stored securely.
+
+The authorization header for Apps is similar to the one used for users:
+
+```http
+POST /graphql/ HTTP/1.1
+...
+Authorization: Bearer <app-token>
+```
+
+### Dashboard extensions
+
+Saleor Apps extending the dashboard will be passed a special JSON Web Token (JWT) with the effective permission set equal to the intersection of user and application permissions. This token allows your frontend part to access the API on behalf of the user.
```

Comment:
you can link https://github.com/saleor/saleor-app-sdk/blob/main/docs/app-bridge.md

Reply:
Please either add the link or submit a PR to add it.

BTW, as a user, I’d expect the SDK to be something covered by the docs, not an external resource.